clusters/hosted-mgmt/hive/pools/rhtap/rhtap-aws-us-west-2.yaml

```diff
@@ -12,8 +12,8 @@ stringData:
       name: worker
       platform:
         aws:
-          type: m5.xlarge
-      replicas: 4
+          type: m6i.2xlarge
```

Comment:
https://instances.vantage.sh/aws/ec2/m5.xlarge vs. https://instances.vantage.sh/aws/ec2/m6i.2xlarge

Reply:
wdyt about https://instances.vantage.sh/aws/ec2/m5.2xlarge?

Reply:
It also has 8 vcpus 32 GB ram per node